analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -385,7 +398,22 @@ class PIP(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfigura
         // Create an out-of-tree virtualenv.
         log.info { "Creating a virtualenv for the '${workingDir.name}' project directory..." }
         val virtualEnvDir = createTempDir(workingDir.name.padEnd(3, '_'), "virtualenv")
-        ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path).requireSuccess()
+
+        //Create virtual environment with correct pip version. Basically we don't know where Pythons were installed.
+        //By default on Linux commands "python" and "python2" are using for Python version 2. After install Python version >3, should be active also command "python3".
+        //On Windows we have to add Python interpreters to environment variables and they should have another names.
+        //See: https://stackoverflow.com/a/22626734/5877109
+        val projectHasPython3 = ProjectVersion.isPython3(workingDir)
+        var virtualEnv = when (Pair(projectHasPython3, OS.isWindows)){
+            Pair(true, true) -> ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path, "-p", "python.exe")
+            Pair(false, true) -> ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path, "-p", "python2.exe")
+            Pair(true, false) -> ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path, "-p", "python3")
+            Pair(false, false) -> ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path, "-p", "python2")
+
+            else -> throw IllegalArgumentException("Unanle to create virtual environment fot Python.")
```

Comment:
There are some typos here 😉

Reply:
OMG, right, terrible, I didn't see it but I've already removed this line because we don't need it ;-) .